akka-docs/rst/java/serialization.rst

```diff
@@ -59,23 +59,18 @@ to disable a default serializer, map its marker type to “none”::
 Verification
 ------------
 
-If you want to verify that your messages are serializable you can enable the following config option:
+Normally, messages sent between local actors (i.e. same JVM) do not undergo serialization. For testing, sometimes, it may be desirable to force serialization on all messages (both remote and local). If you want to do this in order to verify that your messages are serializable you can enable the following config option:
 
 .. includecode:: ../scala/code/docs/serialization/SerializationDocSpec.scala#serialize-messages-config
 
-.. warning::
-
-   We only recommend using the config option turned on when you're running tests.
-   It is completely pointless to have it turned on in other scenarios.
-
 If you want to verify that your ``Props`` are serializable you can enable the following config option:
 
 .. includecode:: ../scala/code/docs/serialization/SerializationDocSpec.scala#serialize-creators-config
 
 .. warning::
 
-   We only recommend using the config option turned on when you're running tests.
-   It is completely pointless to have it turned on in other scenarios.
+   We recommend having these config options turned on **only** when you're running tests. It is completely pointless to have these turned on in other scenarios (e.g. production). The resulting performance loss can be dramatic.
```

Comment:
"completely pointless" sounds very negative (i see we had this wording before), shall we rephrase as "Turning this option on in production is pointless, as it would negatively impact local message passing performance dramatically without giving any gain"?